core/src/main/java/org/infinispan/configuration/cache/BackupConfigurationBuilder.java

```diff
@@ -91,17 +91,16 @@ public TakeOfflineConfigurationBuilder takeOffline() {
    }
 
    /**
-    * Configures how the system behaves when the backup call fails. Only applies to sync backups.
-    * The default values is  {@link org.infinispan.configuration.cache.BackupFailurePolicy#WARN}
+    * Configures how the system behaves when the backup call fails. Only applies to sync backups. The default values is
+    * {@link org.infinispan.configuration.cache.BackupFailurePolicy#WARN}
```

Comment:
```suggestion
    * Configures the policy for handling failed requests for synchronous backups. Defaults to 
    * {@link org.infinispan.configuration.cache.BackupFailurePolicy#WARN}
```

Reply:
I'll remove the word `synchronous` since not it support both sync and async